python/modules/JetSelection.py

```diff
@@ -99,19 +124,44 @@ def analyze(self, event):
             else:
                 setattr(jet,"minDPhiClean",100)
                 setattr(jet,"minDRClean",100)
-                
+            
             selectedJets.append(jet)
             
+        def metP4(obj):
+                    p4 = ROOT.TLorentzVector()
+                    p4.SetPtEtaPhiM(obj.pt,0,obj.phi,0)
+                    return p4
+  
+        #print((metP4(met)).E())
+        self.out.fillBranch("MET_energy", (metP4(met)).E())
+	
+        for outputName, jet_list in zip(self.outputName_list, [selectedJets,unselectedJets]):
+            self.out.fillBranch("n"+outputName, len(jet_list))
+            
+            if self.fatFlag:
+                if not Module.globalOptions["isData"]:
+                    self.out.fillBranch(outputName+"_genJetAK8Idx", map(lambda jet: jet.genJetAK8Idx, jet_list))
```

Comment:
can these properties not be added to 'storeKinematics'? this way it will not be saved by default

Reply:
already taken care of that